extensions/binary.py

```
@@ -15,11 +15,20 @@
-    # CLASS -> BYTES TO STRING
-    def bin2str(self, bytecode: bytes) -> str:
-        return bytecode.hex().upper()
\ No newline at end of file
+    # CLASS -> LOAD
+    def load(self, data: bytes | str) -> None:
+        self.data = data if isinstance(data, bytes) else bytes.fromhex(data)
+    # CLASS -> SERIALIZE
+    def serialize(self) -> str:
+        return self.data.hex().upper()
+    # CLASS -> GET
+    def get(self) -> StreamingResponse:
```

Ruff (0.12.2)

`StreamingResponse` may be undefined, or defined from star imports

(F405)